extract-cert-and-key-from-pfx-and-remove-rsa-password

```diff
@@ -1,7 +1,7 @@
-#!/bin/bash 
+#!/bin/bash
 
 function usage() {
-   echo "Extract the key and certificate from a PFX or P12 file"
+   echo "Extract the key with RSA password removed and certificate from a PFX or P12 file"
```

Comment:
I think you meant to write: "Extract the key and certificate from a PFX or PKCS12 file and remove the passphrase"

Reply:
That sounds better. Fixed!